tests/acceptance/stepDefinitions/notificationsContext.js

```diff
@@ -0,0 +1,25 @@
+const { client } = require('nightwatch-api')
+const { When, Then } = require('cucumber')
+const httpHelper = require('../helpers/httpHelper')
+const fetch = require('node-fetch')
+
+When('the user is sent a notification', function () {
+  const headers = httpHelper.createAuthHeader(client.globals.backend_admin_username)
+  return fetch(
+    client.globals.backend_url + '/ocs/v2.php/apps/testing/api/v1/notifications',
+    { method: 'POST', headers: headers }
+  )
+    .then(res => httpHelper.checkStatus(res, 'Could not generate notification.'))
+})
+
+When('the user clicks on Mark as read', function () {
```

Comment:
don't use specific words like `click` in the description of the step. This should more describe the workflow not the nitty-gritty of the things the user does. The same "steps" have to read nicely if someone reads them e.g. in the context of a mobile device, where no "click" is possible